.github/workflows/build-only.yml

```diff
@@ -0,0 +1,234 @@
+      with:
+        submodules: 'recursive'
+
+    - name: Output versionCode
+      id: versionCode
+      run: |
+        cat mobile/build.gradle | grep versionCode | sed 's/.*\s\([0-9]*\)$/versionCode=\1/' >> "$GITHUB_OUTPUT"
+
+  build-apk:
+    name: Build ${{ matrix.type }}
+    runs-on: ubuntu-20.04
+    needs: [build-rust, get-versionCode]
+    strategy:
+      fail-fast: true
+      matrix:
+        type: ['apk', 'aab']
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+
+    - uses: ActivityWatch/check-version-format-action@v2
+      id: version
+      with:
+        prefix: 'v'
+
+    - name: Echo version
+      run: |
+        echo "${{ steps.version.outputs.full }} (stable: ${{ steps.version.outputs.is_stable }})"
+
+    - name: Set RELEASE
+      run: |
+        # Build in release mode if on a tag/release (longer build times)
+        echo "RELEASE=${{ startsWith(github.ref_name, 'v') }}" >> $GITHUB_ENV
+
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ env.JAVA_VERSION }}
+
+    # Android SDK & NDK
+    - name: Set up Android SDK
+      uses: android-actions/setup-android@v2
+    - name: Set up Android NDK
+      run: |
+        sdkmanager "ndk;${{ env.NDK_VERSION }}"
+        ANDROID_NDK_HOME="$ANDROID_SDK_ROOT/ndk/${{ env.NDK_VERSION }}"
+        ls $ANDROID_NDK_HOME
+        echo "ANDROID_NDK_HOME=$ANDROID_NDK_HOME" >> $GITHUB_ENV
+
+    # Restores jniLibs from cache
+    # `actions/cache/restore` only restores, without saving back in a post-hook
+    - uses: actions/cache/restore@v3
+      id: cache-jniLibs
+      env:
+        cache-name: jniLibs
+      with:
+        path: mobile/src/main/jniLibs/
+        key: ${{ env.cache-name }}-release-${{ env.RELEASE }}-ndk-${{ env.NDK_VERSION }}-${{ hashFiles('.git/modules/aw-server-rust/HEAD') }}
+        fail-on-cache-miss: true
+
+    - name: Check that jniLibs present
+      run: |
+        test -e mobile/src/main/jniLibs/x86_64/libaw_server.so
+
+    - name: Set versionName
+      if: startsWith(github.ref, 'refs/tags/v')  # only on runs triggered from tag
+      run: |
+        # Sets versionName, tail used to skip "v" at start of tag name
+        SHORT_VERSION=$(echo "${{ github.ref_name }}" | tail -c +2 -)
+        sed -i "s/versionName \".*\"/versionName \"$SHORT_VERSION\"/g" \
+                mobile/build.gradle
+
+    - name: Set versionCode
+      run: |
+        # Sets versionCode
+        sed -i "s/versionCode .*/versionCode ${{needs.get-versionCode.outputs.versionCode}}/" \
+                mobile/build.gradle
+
+    - uses: adnsio/setup-age-action@v1.2.0
+    - name: Load Android secrets
+      if: env.KEY_ANDROID_JKS != null
+      env:
+        KEY_ANDROID_JKS: ${{ secrets.KEY_ANDROID_JKS }}
+      run: |
+        printf "$KEY_ANDROID_JKS" > android.jks.key
+        cat android.jks.age | age -d -i android.jks.key -o android.jks
+        rm android.jks.key
```

Comment:
What does this do? I don't understand where `android.jks.age` is initially created.